Makefile

```diff
@@ -9,6 +10,16 @@ ifndef ZJS_BASE
 $(error ZJS_BASE not defined. You need to source zjs-env.sh)
 endif
 
+# SIZE can't be less than the 144KB physically allocated for x86
+ifeq ($(shell test $(SIZE) -lt 144; echo $$?), 0)
+$(error SIZE must be at least 144)
```

Comment:
We haven't been doing that at the file scope, if you look around. In Emacs at least it defaults to using Tab in a Makefile, and if you indent with Tab here it breaks it because that has special meaning. Apparently if you type four spaces manually it does work, but it doesn't seem like too great an idea to have a mixture of things indented with spaces and tabs in the same file?